consensus/core/src/linearizer.rs

```diff
@@ -287,11 +279,7 @@ impl Linearizer {
         // Sort the blocks of the sub-dag blocks
         sort_sub_dag_blocks(&mut to_commit);
 
-        // TODO(fastpath): determine rejected transactions from voting.
```

Comment:
Determining rejected transactions is done in CommitFinalizer later, outside of Core loop.